src/Directory.Build.props

```diff
@@ -1,16 +1,3 @@
 <Project>
   <Import Project="../Directory.Build.props" />
-
-  <!-- Properties that should be defined for all managed product assemblies -->
```

Comment:
These are for the product itself, not for the repo. These settings should stay here.

Reply:
Partially agree. I can move them back here and let it only affect product build. Some things might have value in being labeled if we plan to package our tests for external consumers. Although the more I think, I like @jkotas observation of following suit and letting it be in Version.props for cross repo consistency and also because ideally after moving all our assets to an arcade build, it would be nice to have one source of truth for versioning details.

Reply:
I am fine with them in `eng/` or even in a new sub-folder. My point here is that the root of the repo should only contain universal build settings that are shared between product and test. If the setting isn't needed by both builds it should not exist at the root.

Reply:
Sounds fine with me :) I'll make the change then.